pkg/operator/workloads/api_workload.go

```diff
@@ -275,9 +275,29 @@ func tfAPISpec(
 				"userFacing":   "true",
 			},
 			Annotations: map[string]string{
-				"sidecar.istio.io/inject": "true",
+				"traffic.sidecar.istio.io/excludeOutboundIPRanges": "0.0.0.0/0",
```

Comment:
Reading through the [docs](https://istio.io/docs/setup/kubernetes/additional-setup/cni/#traffic-redirection-parameters) on this again, this annotation might not be necessary. Can you try without and see if it works?